packages/deployer/test/fixture-projects/sample-project/test/UpgradeModule.test.js

```diff
@@ -86,7 +86,7 @@ describe('UpgradeModule', () => {
     });
   });
 
-  describe('when attempting to destroy the implementation with a malicious contract', () => {
```

Comment:
skipped since the anti-bricking protection is disabled. Will be set back (and moved to core-contracts) with issue 214